app/models/ingest_job.rb

```diff
@@ -391,9 +394,20 @@ def set_subsampling_flags
   #   - (Boolean) => True/False on success of file copy action
   def create_study_file_copy
     begin
-      ApplicationController.firecloud_client.execute_gcloud_method(:copy_workspace_file, 0, self.study.bucket_id,
-                                                   self.study_file.bucket_location,
-                                                   self.study_file.parse_fail_bucket_location)
+      ApplicationController.firecloud_client.execute_gcloud_method(:copy_workspace_file, 0,
+                                                                   self.study.bucket_id,
+                                                                   self.study_file.bucket_location,
+                                                                   self.study_file.parse_fail_bucket_location)
+      if self.study_file.is_bundled? && self.study_file.is_bundle_parent?
```

Comment:
Thanks for noticing and adding to this bugfix that we'd also want to cache the associated bundle files on ingest failure!